advarchs/__init__.py

```diff
@@ -27,9 +27,11 @@
     else:
         raise AdvarchsException('Unpacker is not installed.')
 elif os.name == 'posix':
-    if (shutil.which('7z')) and (shutil.which('unrar')):
-        HandlersFactory.register('7z', SevenZHandler('7z'))
+    if (shutil.which('7z') or shutil.which('7za')) and (shutil.which('unrar')):
```

Comment:
consider code optimization
sevenz = max([a if which(a) else None for a in ('7z', '7za')], key=lambda x: bool(x), default=None)

# elif os.name == 'posix':
#     if (sevenz) and (shutil.which('unrar')):
#         HandlersFactory.register(sevenz)
#         HandlersFactory.register('unrar', RarHandler('unrar'))
#     elif (sevenz) and not (shutil.which('unrar'))):
#         HandlersFactory.register(sevenz)
#     else:
#         raise AdvarchsException('Unpacker is not installed.')